Rules/Rules.csproj

```diff
@@ -1,79 +1,80 @@
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="RulesMod.cs" />
+    <Compile Include="Rule\AbilityActionCostAdjustedRule.cs" />
+    <Compile Include="Rule\AbilityDamageAdjustedRule.cs" />
+    <Compile Include="Rule\ActionPointsAdjustedRule.cs" />
+    <Compile Include="Rule\CardEnergyFromAttackMultipliedRule.cs" />
+    <Compile Include="Rule\CardEnergyFromRecyclingMultipliedRule.cs" />
+    <Compile Include="Rule\CardSellValueMultipliedRule.cs" />
+    <Compile Include="Rule\EnemyAttackScaledRule.cs" />
+    <Compile Include="Rule\EnemyDoorOpeningDisabledRule.cs" />
+    <Compile Include="Rule\EnemyHealthScaledRule.cs" />
+    <Compile Include="Rule\EnemyRespawnDisabledRule.cs" />
+    <Compile Include="Rule\GoldPickedUpMultipliedRule.cs" />
+    <Compile Include="Rule\RatNestsSpawnGoldRule.cs" />
+    <Compile Include="Rule\SampleRule.cs" />
+    <Compile Include="Rule\PieceConfigAdjustedRule.cs" />
+    <Compile Include="Rule\StartHealthAdjustedRule.cs" />
+    <Compile Include="Rule\ZapStartingInventoryAdjustedRule.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Assembly-CSharp">
+      <HintPath>$(SteamDemeoDir)\demeo_Data\Managed\Assembly-CSharp.dll</HintPath>
+    </Reference>
+    <Reference Include="MelonLoader">
+      <HintPath>$(SteamDemeoDir)\MelonLoader\MelonLoader.dll</HintPath>
+    </Reference>
+    <Reference Include="System" />
+    <Reference Include="UnityEngine.CoreModule">
+      <HintPath>$(SteamDemeoDir)\demeo_Data\Managed\UnityEngine.CoreModule.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\RulesAPI\RulesAPI.csproj">
+      <Project>{b412463f-a4f7-46ce-ac4e-37448762b45b}</Project>
+      <Name>RulesAPI</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="README.md" />
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Target Name="CopyToModsDir" AfterTargets="Build">
+    <Copy SourceFiles="$(OutputPath)\$(ProjectName).dll" DestinationFolder="$(SteamDemeoDir)\Mods" />
+    <Copy SourceFiles="$(OutputPath)\$(ProjectName).dll" DestinationFolder="$(OculusDemeoDir)\Mods" />
+  </Target>
+</Project>
```

Comment:
https://stackoverflow.com/a/38556923  😅

Reply:
The Rules.csprod isn't being edited by me... It's happening automatically inside Visual Studio (not VSCode). I use VSCode for my day-to-day stuff, but I'm using VS(non-code) for the demo modding stuff because I don't know any better. Any idea how to stop Visual Studio from getting it wrong?

Reply:
I use JetBrains Rider, but it looks like these are the steps that should work for VS:
1. Open the `.csproj` file: https://stackoverflow.com/a/64428453
2. Bottom-right of the screen, change indents: https://stackoverflow.com/a/38556923

Not urgent, of course 👍🏾

Reply:
The first step was OK... the second step sadly not. Visual Studio has the tabs/spaces settings buried in an options menu, and every one of them is set to 4-spaces and 'insert spaces'. They are different settings for different languages/filetypes and the setting for XML is already on 4-spaces - there is no filetype listed for .csprod files. 

Found a couple of posts along the same lines as the following one. No clear answers though.
https://stackoverflow.com/questions/68015170/visual-studio-web-csproj-file-indenting-flipping-between-tabs-and-spaces.

I manually fixed the .csproj for the AOE PR, and was about to fix this one too, but you merged it before I got the chance.